xbdd-web/src/modules/report/FeatureListContainer/TagViewFeatureList/TagViewFeatureList.jsx

```diff
@@ -15,20 +15,20 @@ const TagListFeaturesListView = props => {
 
   return (
     <List className={classes.xbddFeatureListContainer}>
-      {featureList.map(item => {
+      {featureList.map(feature => {
         return (
-          <ListItem button key={item.id} className={classesMap[item.calculatedStatus]}>
-            {item.name}
+          <ListItem button key={feature.id} className={classesMap[feature.calculatedStatus] + " " + classes.xbddTagViewFeatureList}>
```

Comment:
Can we use the backtick notation here for consistency? i.e. \`${classesMap[feature.calculatedStatus]} ${classes.xbddTagViewFeatureList}\`